.circleci/config.yml

```diff
@@ -43,10 +61,68 @@ jobs:
           key: uikit-yarn-dependencies-{{ checksum "yarn.lock" }}
           paths:
             - ./.yarn/cache
+  create-ticket:
+    macos:
+      xcode: "14.1"
+    steps:
+      - checkout
+      - clone_deployment_script
+      - run:
+          name: Extract release version
+          command: |
+            release_version=$(python3 ~/sdk-deployment/scripts/v1.2/extract_version.py --branch_name $CIRCLE_BRANCH)
```

Comment:
need small clarification, is this gonna be `release/v3.2.4` or just `release/3.2.4`?
good to comment here

Reply:
I set a validation to make the version name start with `release/v`. 
The gh workflow will only be executed when the branch name starts wth `release/v`. 
But I probably need to add a guidance to let ppl know this in some say 
https://github.com/sendbird/sendbird-uikit-react/pull/658/commits/49255ab8b33b8144b9dffda32553bad820eb3c8d